scripts/get_experiments_data.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#Use from main directory (opening-black-box)
+path=$(pwd)
+filename="experiments_data.zip"
+#dockr -exec -it psql -d black-box -t -A -F"," -c "select * from stage_metrics" > stage_metrics_csv
+rm -rf experiments_data
```

Comment:
Do we want to remove this directory every time we download the data?

Reply:
every new data should IMO override the old data (as far as we keep it locally)